easyconfigs/d/DCC/DCC-0.5.0-foss-2021b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'DCC'
+version = '0.5.0'
+
+homepage = 'https://github.com/dieterich-lab/DCC'
+description = "DCC is a python package intended to detect and quantify circRNAs with high specificity. DCC works with the STAR (Dobin et al., 2013) chimeric.out.junction files which contains chimerically aligned reads including circRNA junction spanning reads."
```

Comment:
I can't quite see in GitHub but I think this probably exceeds the 120 character line limit. Also, we normally use triple quotes (`"""`)for all descriptions so that it's easier to span multiple lines. Have a quick look at some of the other descriptions. (When breaking a line we always put the space at the beginning of the new line.)